src/pdf/ShoppingList/dataPreparation.ts

```diff
@@ -108,10 +112,10 @@ const getClientAndFamilyData = async (clientID: string): Promise<ClientDataAndFa
     return { clientData: clientData, familyData: familyData };
 };
 
-const processExtraInformation = (original: string): NappySizeAndExtraInformation => {
+export const processExtraInformation = (original: string): NappySizeAndExtraInformation => {
     if (original.startsWith("Nappy Size: ")) {
-        const [nappySize, extraInformation] = original.split(", Extra Information: ", 1);
-        return { nappySize: `(${nappySize})`, extraInformation: extraInformation };
+        const [nappySize, extraInformation] = original.split(", Extra Information: ");
```

Comment:
`, 1` has been removed here

Reply:
yep, it's because , 1 will limit the returned things to be 1 element in the list - so extraInformation will be undefined